test/graphql/types/Organization/advertisement.test.ts

```
@@ -0,0 +1,661 @@
+
+            const secondPage = secondPageResult.data?.organization?.advertisements;
+            assertToBeNonNullish(secondPage);
+            expect(secondPage.edges).toHaveLength(2);
+            expect(secondPage.pageInfo.hasNextPage).toBe(true);
+
+            const firstPageAdNames = firstPage.edges
+                ?.filter(edge => edge?.node != null)
+                .map(edge => edge?.node?.name) ?? [];
+            const secondPageAdNames = secondPage.edges
+                ?.filter(edge => edge?.node != null)
+                .map(edge => edge?.node!.name) ?? [];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)